h2o-algos/src/main/java/hex/gam/GAMModel.java

```diff
@@ -238,8 +238,6 @@ public long progressUnits() {
       return 1;
     }
 
-    public long _seed = -1;
```

Comment:
This `_seed` shadows the seed from model builder. Changing this `_seed` caused changes only for the GAM class but in the ancestral classes it used to use different seed that was unaffected when this seed changed. This led to some indeterminism (it took some time more iterations to come to the same result which might have not been noticeable until limiting the `max_iteration` it).